legal-api/tests/unit/services/test_document_meta.py

```diff
@@ -16,13 +16,61 @@
 
 Test-Suite to ensure that the Document Meta Service is working as expected.
 """
-
+from legal_api.models import Business
 from legal_api.services import document_meta
-from legal_api.utils.legislation_datetime import LegislationDatetime
 
 
-def test_filing_status_documents_empty(app):
-    """Assert that no documents are returned for non PAID and COMPLETED filing."""
+def factory_business(identifier: str, legal_type: str = ''):
+    """Return a valid Business object stamped with the supplied data."""
+    business = Business(identifier=identifier, legal_type=legal_type)
+    business.save()
+    return business
+
+
+def test_business_factory(session):
+    """Assert that factory_business works correctly."""
+    business = factory_business('BC1234567', 'BC')
+    assert business.id is not None
+    assert business.identifier == 'BC1234567'
+    assert business.legal_type == 'BC'
+
+
+def test_find_by_identifier(session):
+    """Assert that find_by_identifier works correctly."""
```

Comment:
Not sure why this test is part of document meta service. Similar tests are already there for business model

Reply:
This was very helpful for me to build up the unit tests and verify that each step worked.
But you are right... fixed in next commit.